src/olympia/reviewers/views.py

```diff
@@ -62,7 +62,7 @@ def context(request, **kw):
     return ctx
 
 
-@addons_reviewer_required
+@any_reviewer_required
```

Comment:
This will expose the add-on review log to users who don't have access to view the review pages.

Reply:
Yeah. https://github.com/mozilla/addons-server/issues/6522 asks (in the linked doc) to put a link to the event log for users with ratings moderation permission, and they might not have anything else, so...

Reply:
this view method is for the review log, rather than moderation log, isn't it?  (The naming is hella confusin)

Reply:
It confused me as well... `eventlog` is different than `reviewlog`. The former calls `ActivityLog.objects.reviewer_events()` and that only fetches `ActivityLog` actions that deal with ratings (it uses `LOG_REVIEWERS` which uses `reviewer_event` which is only set for `APPROVE_RATING` and `DELETE_RATING`).

I'll file an issue to rename it - but in any case, it seems like a good candidate for `any_reviewer_required`, because I kinda consider that ratings moderation is the "lowest" permission you can get.

Reply:
Oh, yeah.  So it should be next to the `reviewlog` function below.  I'm reading this as we're giving access to the review history, the highest access, (potentially the most sensative, because it might detail unlisted or admin reviewed add-ons) to the users with the lowest permission.

Reply:
I filed https://github.com/mozilla/addons-server/issues/6931